install_config/aggregate_logging.adoc

```diff
@@ -442,6 +442,17 @@ remove all logging persistent data including PVC, set
 `openshift_logging_install_logging` to 'false' to trigger uninstallation and
 `openshift_logging_purge_logging` to 'true'. The default is set to 'false'.
 
+|`openshift_logging_install_logging`
```

Comment:
Few small details, we already have `openshift_logging_install_logging` documented on top of the page, I think that description up there should suffice.

And would you consider moving `openshift_logging_purge_logging` and `openshift_logging_install_eventrouter` right under the original description of `openshift_logging_install_logging` on top of this page? These three are tied and could influence one another, and I think it would make sense to have them together

Reply:
Ah yes, you're right. I moved them together and will update the PR.